### DISCUSSION TITLE
How to use these features of highlighted text?

### DISCUSSION THREAD

**Discussion by stvivec** [0x7c13/Notepads#1107] [Q&A] [answered]
Just saw them on the marketplace and was wondering how to use them.
![image](https://user-images.githubusercontent.com/105086596/167227195-28db74b3-e002-4c33-80ea-c1076d54d4b5.png)

**Comment by 0x7c13** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/1166162/167337486-b7ea7bea-d685-41fd-977b-37bce0caeb80.png)
